### PR TITLE
Force option for remove-application and remove-unit commands.

### PR DESCRIPTION
**Pull request by anastasiamac**
## Description of change

Juju is introducing --force option for remove-application and remove-unit commands.
This is the first PR in the series. It simply adds the option to the commands and ensure that it is passed in to the api and "Application" client facade.

### CODE REVIEW

**Comment by anastasiamac** [comment]
[add force option to remove-application and remove-unit commands](https://trello.com/c/cydoSzuk/659-add-force-option-to-remove-application-and-remove-unit-commands)

File: apiserver/allfacades.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func AllFacades() *facade.Registry {
 	reg("Application", 6, application.NewFacadeV6)
 	reg("Application", 7, application.NewFacadeV7)
 	reg("Application", 8, application.NewFacadeV8)
-	reg("Application", 9, application.NewFacadeV9) // ApplicationInfo, generational config.
+	reg("Application", 9, application.NewFacadeV9) // ApplicationInfo, generational config, Force on App and Unit Removal.
```
Comment (babbageclunk) [inline apiserver/allfacades.go]: I love it when I need to add something to an API and it's already been version-bumped for me!

**Comment by anastasiamac** [comment]
$$merge$$

File: apiserver/allfacades.go (hunk 1 above)
Comment (anastasiamac) [reply apiserver/allfacades.go]: Me too! Thank you, team :D

**Comment by anastasiamac** [comment]
unrelated failure in github.com/juju/juju/apiserver/facades/agent/fanconfigurer.TestAll

**Comment by anastasiamac** [comment]
$$merge$$

**Pull request merged**